Gems/GraphCanvas/Code/StaticLib/GraphCanvas/Widgets/NodePalette/NodePaletteWidget.cpp

```diff
@@ -64,27 +64,6 @@ namespace GraphCanvas
             QModelIndex sourceIndex = static_cast<const NodePaletteSortFilterProxyModel*>(index.model())->mapToSource(index);
             NodePaletteTreeItem* treeItem = static_cast<NodePaletteTreeItem*>(sourceIndex.internalPointer());
 
-            if (treeItem)
-            {
-                // Make the text slightly transparent if the item is disabled
-                if (!treeItem->IsEnabled())
```

Comment:
This logic draws the text in the tree view. I move it right after the background box-highlight drawing (ElidingLabel::paintEvent also does so in this order). I had to make the change given that since the move to standalone, the background highlight would be drawn after the text thus hiding it completely.

I haven't looked further as to why it behaves differently in standalone given it likely worked in the past because of a wrong cached state in the painter.